dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -167,8 +160,8 @@ private void processBody() throws IOException {
         requiredLength = lengthFieldOffset + lengthFieldLength;
     }
 
-    protected void invokeListener(InputStream inputStream) {
-        this.listener.onFragmentMessage(dataHeader, inputStream);
+    public void invokeListener(InputStream inputStream) {
+        this.listener.onFragmentMessage(inputStream);
```

Comment:
**Issue**: Changing the visibility of invokeListener from protected to public may expose internal mechanisms unnecessarily. <br> **Fix**: Review the necessity of this visibility change. If external access is not needed, revert to protected. <br> **Code Suggestion**: 
 ```
 -    protected void invokeListener(InputStream inputStream) {
 +    private void invokeListener(InputStream inputStream) {
 ```